lib/trento_web/controllers/fallback_controller.ex

```diff
@@ -138,6 +138,13 @@ defmodule TrentoWeb.FallbackController do
     |> render(:"403")
   end
 
+  def call(conn, {:error, :forbidden}) do
```

Comment:
I would personally remove the `operation_not_permitted` from above, and only have one.

Reply:
yes, done and refactored